pages/start/scaffold.mdx

```diff
@@ -44,19 +44,19 @@ Open [http://localhost:5173/](http://localhost:5173/) in two tabs to see it in a
 When you are happy with your app, publish the Reflect server to [reflect.net](https://reflect.net/):
 
 ```bash copy
-npx reflect publish
+npx reflect publish --app my-app
```

Comment:
What is the current state of defaulting of the `--app` flag here. Does it do any defaulting if not specified or is it completely required?

Reply:
it is completely required now unless it an appid is defined in reflect.config.json. we had a customer run publish and was surprised it was defaulting to the name of the app in package.json so that is now removed.